### ISSUE TITLE
Missing learning rate scheduler in RNN LM training in the librispeech recipe

### ISSUE THREAD

**Issue by csukuangfj** [espnet/espnet#3834]
There is no learning rate scheduler from the following LM config:
https://github.com/espnet/espnet/blob/9827dfe37f69e8e55f902dc4e340de5108596311/egs/librispeech/asr1/conf/tuning/lm_large.yaml#L1-L9

Also, the default value for the learning rate scheduler is `None`:
https://github.com/espnet/espnet/blob/9827dfe37f69e8e55f902dc4e340de5108596311/espnet/bin/lm_train.py#L158-L166

---

However, there is a learning rate scheduler in the transformer-based LM training, see
https://github.com/espnet/espnet/blob/9827dfe37f69e8e55f902dc4e340de5108596311/egs/librispeech/asr1/conf/tuning/lm_transformer.yaml#L21

Is there any reason to not use LR scheduler in RNN LM training?

Also CC @danpovey

**Comment by sw005320** [comment]
@ShigekiKarita, can you answer it?

**Comment by ShigekiKarita** [comment]
This is simply because of differences in reference implementations. IIRC, Transformer LM and LR scheduler were based on https://github.com/NVIDIA/Megatron-LM#gpt-pretraining. On the other hand, the LSTM LM config was based on some configs in https://github.com/rwth-i6/returnn

As it has been outdated, adding LR schedulers sounds good to me :+1:
